app/views/admin/dashboard/index.html.erb

```diff
@@ -4,4 +4,41 @@
       <h1 class="green-font mousiness" >Admin Dashboard</h1>
     </div>
   </div>
+  <div class="text-center row">
+    <div class="padded-bottom">
+      <h2 class="green-font mousiness" >Order Statuses</h1>
+    </div>
+  </div>
+  <div class="orders row">
+    <ul>
+      <li><%= link_to "Ordered", admin_status_path(status: "ordered")%>: <%= @orders.status_total("ordered") %></li>
+      <li><%= link_to "Cancelled", admin_status_path(status: "cancelled")%>: <%= @orders.status_total("cancelled") %></li>
+      <li><%= link_to "Paid", admin_status_path(status: "paid")%>: <%= @orders.status_total("paid") %></li>
+      <li><%= link_to "Completed", admin_status_path(status: "completed")%>: <%= @orders.status_total("completed") %></li>
+    </ul>
+    <table class="table table-condensed table-hover row">
+      <tr>
+        <th>Order #</th>
+        <th>Status</th>
+        <th>Placed on</th>
+        <th>Update Order</th>
+      </tr>
+      <% @orders.each do |order| %>
+        <tr>
+          <td><%= link_to "##{order.id}", order_path(order.id) %></td>
+          <td><%= order.status %></td>
+          <td><%= order.created_at.strftime("%B %-d, %Y") %></td>
+          <% if order.status == "paid" %>
```

Comment:
I'm going to leave this in for right now, it's something that will clearly get taken care of when we refactor.